### DISCUSSION TITLE
Data Migration issue

### DISCUSSION THREAD

**Discussion by tharung07** [minio/minio#19456] [Q&A]
Hi Team,

We are facing errors while migrating data through MC command from local system to minio bucket.

Could you please provide solution to this problem and also find error image below : 
![image](https://github.com/minio/minio/assets/165164157/6244c896-1383-4a7c-a479-76382f3dc6df) 

Regards,
Tharun Chary

**Comment by klauspost** [comment]
There is no problem showing with what you are posting.

**Reply by tharung07** [reply]
It shows a '404 Not Found' error when the data migration is stopped, and the disk in Ubuntu goes offline. After some time, it comes back online again.

**Reply by klauspost** [reply]
It is normal for BucketTagging and BucketPolicy to return 404 if they haven't been set.
